### ISSUE TITLE
Resources subfolders are not copied/served when using Rollup

### ISSUE THREAD

**Issue by creal73** [muenzpraeger/create-lwc-app#627]
### Summary

Resources subfolders are not copied into dist when using Rollup

### Steps To Reproduce:

1.  `npx create-lwc-app`
1.  Do you want to use the simple setup? **No**
1. Package name for npm **test**
1. Description
1. Author
1. Select a package manager **npm**
1. Select the type of app you want to create **Progressive Web App (PWA)**
1. Select a bundler **Rollup**
1. Use TypeScript or JavaScript **JavaScript**
1. Do you want a basic Express API server? **No**
1. `npm run watch`

### Expected result

Subfolders (e.g icons) should be copied into dist and served.

### Actual result

Subfolders (e.g icons) are not copied into dist and so, not served.

### Additional information

![image](https://user-images.githubusercontent.com/587284/86577424-7e0e4080-bf7a-11ea-99d0-70a4eebb3251.png)

![image](https://user-images.githubusercontent.com/587284/86577997-67b4b480-bf7b-11ea-845c-e81631001aa0.png)

**Comment by rochejul** [comment]
+1

**Comment by muenzpraeger** [comment]
Fixed, and published with 2.1.4 - thanks for the finding!

For existing projects you can just change the glob pattern in `lwc-services.config.js` by adding a second wildcard param to the `from` key:

```js
module.exports = {
    resources: [
        { from: 'src/resources/**', to: 'dist/resources/' }
    ]
};
```
